tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "allowJs": true,
     "checkJs": true,
     "isolatedModules": true,
-    "types": ["vite/client"]
+    "types": ["vite/client"],
+    "strict": false
```

Comment:
No change here; strict=false is the default. Just adding this to make it slightly easier to toggle to true temporarily. We're _almost_ at the point of enabling strict for real! #89